src/components/Notification/InactivityMessage.jsx

```diff
@@ -58,25 +61,54 @@ const InactivityMessage = () => {
     };
   }, []);
 
+  /* Starts a five minute timer when the inactivity window pops up. 
+     Logs the user out and refreshes the page if time elapses. */
+  useEffect(() => {
+    if (showPopup) {
+      logoutTimer.current = setInterval(() => {
+        if (seconds === 0 && minutes === 0) {
+          handleLogout();
+          window.location.reload(false);
+        } else if (seconds === 0) {
+          setSeconds(59);
+          setMinutes((minutes) => minutes - 1);
+        } 
+          else if (seconds > 0) {
+          setSeconds((seconds) => seconds - 1);
+        }
+      }, 1000)
+    }
+    return () => clearInterval(logoutTimer.current);
+  }, [showPopup, seconds, minutes])
+
   // Event handler for logout and removing items from localStorage
   // Returns user to home page upon successful logout
   const handleLogout = () => {
     localStorage.clear();
   };
 
+  // Event handler for popup's continue button
+  // Closes the window, and resets the timer
+  const handleClosePopup = () => {
+    setShowPopup(false);
+    setSeconds(0);
```

Comment:
For the set functions for the time, I think we could have them placed inside the clean-up function for the useEffect with the logoutTimer to reset the time rather than inside a different handler function.

Reply:
Ok, I moved them there and removed the "handleClosePopup" function.